ppr-ui/src/components/userAccess/ReviewConfirm/QsInformationReview.vue

```diff
@@ -1,32 +1,44 @@
 <template>
   <PartyReview
     :baseParty="getMhrQsInformation"
-    :showIncomplete="!getMhrUserAccessValidation.qsInformationValid"
+    :showIncomplete="!getMhrUserAccessValidation.qsInformationValid || !getMhrUserAccessValidation.qsSaConfirmValid"
     :returnToRoutes="[RouteNames.QS_USER_ACCESS, RouteNames.QS_ACCESS_INFORMATION]"
   >
     <!-- Header Override -->
-    <template v-slot:headerSlot>
+    <template #headerSlot>
       <header class="review-header">
         <v-icon class="ml-1" color="darkBlue">mdi-account-lock</v-icon>
         <label class="font-weight-bold pl-2">Qualified Supplier ({{ getMhrSubProduct }}) Information</label>
       </header>
     </template>
 
+    <template #partyInfoLabelSlot>
+      <v-row no-gutters class="px-8 pt-6 mb-n2">
+        <v-col cols="2">
+          <label class="generic-label">Qualified Supplier</label>
+        </v-col>
+      </v-row>
+    </template>
+
     <!-- Conditional based on service agreement checkbox -->
-    <template v-slot:topInfoSlot>
-      <p class="icon-text ml-7 pb-2 mt-8">
-        <v-icon color="success" class="pr-2">mdi-check</v-icon>
-        I have read, understood and agree to the terms and conditions of the Qualified Suppliers’ Agreement for the
-        Manufactured Home Registry.
-      </p>
+    <template v-if="getMhrUserAccessValidation.qsSaConfirmValid" v-slot:topInfoSlot>
```

Comment:
Can update to `#` to match other slot usage in file.

Reply:
Good call, will do.
This was on your recommendation to shorthand, i think we all get it now 👍

Reply:
Updated